tests/Model/StubsContainer.php

```diff
@@ -126,6 +126,11 @@ public function getFunction($name, $sourceFilePath = null, $shouldSuitCurrentPhp
         if (!empty($functions)) {
             return array_pop($functions);
         }
+        // Added this while tracking down "Attempt to read property "parameters" on null" error.
+        // Happens when BasePHPElement::entitySuitsCurrentPhpVersion returns false when running tests on PHP beta: getenv('PHP_VERSION')
+        if (!$functions) {
```

Comment:
I think we can move this check to the `else` branch of the `if (!empty($functions))` above